src/main/resources/reference-databases.xml

```diff
@@ -435,5 +435,87 @@
 				<!-- NOTE: No identifiers.org entry for TargetPathogen -->
 			</map>
 		</entry>
+
+		<entry key="HPA">
+			<map key-type="java.lang.String" value-type="java.lang.String">
+				<entry key="PrimaryName" value="HPA" />
+				<entry key="Aliases">
+					<list value-type="java.lang.String">
+						<value>The Human Protein Atlas</value>
+					</list>
+				</entry>
+				<entry key="URL" value="https://www.proteinatlas.org/" />
+				<entry key="AccessURL" value="https://www.proteinatlas.org/###ID###" />
+			</map>
+		</entry>
+
+		<entry key="VGNC">
+			<map key-type="java.lang.String" value-type="java.lang.String">
+				<entry key="PrimaryName" value="VGNC" />
+				<entry key="Aliases">
+					<list value-type="java.lang.String">
+						<value>Vertebrate Gene Nomenclature Committee</value>
+					</list>
+				</entry>
+				<entry key="URL" value="https://vertebrate.genenames.org/" />
+				<entry key="AccessURL" value="https://vertebrate.genenames.org/data/gene-symbol-report/#!/vgnc_id/VGNC:###ID###" />
+			</map>
+		</entry>
+
+		<entry key="MGI">
+			<map key-type="java.lang.String" value-type="java.lang.String">
+				<entry key="PrimaryName" value="MGI" />
+				<entry key="Aliases">
+					<list value-type="java.lang.String">
+						<value>Mouse Genome Informatics</value>
+					</list>
+				</entry>
+				<entry key="URL" value="http://www.informatics.jax.org/" />
+				<entry key="AccessURL" value="http://www.informatics.jax.org/marker/MGI:###ID###" />
+			</map>
+		</entry>
+
+		<entry key="RGD">
+			<map key-type="java.lang.String" value-type="java.lang.String">
+				<entry key="PrimaryName" value="RGD" />
+				<entry key="Aliases">
+					<list value-type="java.lang.String">
+						<value>Rat Genome Database</value>
+					</list>
+				</entry>
+				<entry key="URL" value="https://rgd.mcw.edu/wg/" />
+				<entry key="AccessURL" value="https://rgd.mcw.edu/rgdweb/report/gene/main.html?id=###ID###" />
+			</map>
+		</entry>
+
+		<entry key="Xenbase">
+			<map key-type="java.lang.String" value-type="java.lang.String">
+				<entry key="PrimaryName" value="Xenbase" />
+				<entry key="URL" value="http://www.xenbase.org/" />
+				<entry key="AccessURL" value="http://www.xenbase.org/gene/showgene.do?method=display&amp;geneId=###ID###" />
+				<!-- NOTE: No identifiers.org entry for OpenTargets or TargetValidation -->
```

Comment:
Did identifiers.org have an entry for Xenbase? Or any of the other new ReferenceDatabases?

Reply:
I didn't look, I just found the working URLs from the resources. I'll look into how identifiers.org does each resource, if available

Reply:
Identifiers.org MIR thingies existed for all except VGNC, thanks!

Reply:
That happens sometimes. *shrug*
 I just leave a note when it does (there are other examples of that in the file).